orchestration/src/main/resources/spec/orchestration.yaml

```diff
@@ -455,20 +543,22 @@ components:
             - azure_content_safety
           example: azure_content_safety
         config:
-          $ref: '#/components/schemas/AzureContentSafety'
+          $ref: "#/components/schemas/AzureContentSafety"
+
     AzureContentSafety:
       description: Filter configuration for Azure Content Safety
       type: object
       additionalProperties: false
       properties:
-        Hate:
-          $ref: '#/components/schemas/AzureThreshold'
-        SelfHarm:
-          $ref: '#/components/schemas/AzureThreshold'
-        Sexual:
-          $ref: '#/components/schemas/AzureThreshold'
-        Violence:
-          $ref: '#/components/schemas/AzureThreshold'
+        "Hate":
+          $ref: "#/components/schemas/AzureThreshold"
+        "SelfHarm":
+          $ref: "#/components/schemas/AzureThreshold"
+        "Sexual":
+          $ref: "#/components/schemas/AzureThreshold"
+        "Violence":
```

Comment:
(Comment)

I'm wondering what makes them using `"` for property keys here, but nowhere else.

Reply:
My best guess is "historical reasons". git blame tells me this is from (one of) the original commits on content filtering 8 months ago.

Reply:
Here's a PR that addresses this: https://github.tools.sap/AI/llm-orchestration/pull/832
(code generation seems unaffected)